tests/js/app-tests.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+"use strict";
```

Comment:
Why did all of the strings change from `'` to `"`?

Reply:
This happened because [the project defines an .eslintrc.json prescribing double quotes](https://github.com/realm/realm-js/blob/master/.eslintrc.json#L19) and I happen to be running VSCode with the eslint plugin formatting on saving.

But - I'll revert and save without formatting and leave the job of fixing this across the codebase (or updating the eslint.json) to a later time.